.github/workflows/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ ] Hi there! There is a name error in this directory which makes your linters check not show up on Github. Kindly make the corrections as the name for this directory is `.github/workflows/linters.yml` and not `.github/workflows/workflows/linters.yml` 😄.